packages/test/src/test-integration-update-signal-query-example.ts

```diff
@@ -0,0 +1,40 @@
+import * as wf from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers-integration';
+import {
+  counterWorkflow,
+  getValueQuery,
+  incrementAndGetValueUpdate,
+  incrementSignal,
+} from './workflows/update-signal-query-example';
+
+const test = makeTestFunction({
+  workflowsPath: require.resolve('./workflows/update-signal-query-example'),
```

Comment:
I put it there because it's used in documentation, and that's where the other workflows used in documentation are, and I was worried about breaking documentation. I'll colocate them.